content/blog/esp32-bootstrapping/index.md

```diff
@@ -347,10 +395,80 @@ I (205) heap_runtime: ESP heap runtime init at 0x3fca4f70 size 273 kB.
 uart:~$
 ```
 
+
 ### MCUboot Espressif port (EP)
 
 To learn how to build the MCUboot Espressif Port, check out this [article](https://docs.mcuboot.com/readme-espressif.html)
 
+
+### AMP enabled sample code
+
+AMP builds require several images to be build and flash into a target SoC. Lets use ESP32-S3 as out test platform and demonstrate the sysbuild capabilities on the sample code that is provided by the Zephyr RTOS sources.
+The sample code uses IPM (Inter-Processor Mailbox) to demonstrate simple two way communication between the PROCPU core and APPCPU core. Images for both CPU cores must be loaded using the MCUboot. Note, there is no support to run AMP while using simple boot mechanism.
+
+```shell
+cd zephyrproject/zephyr
+west build -b esp32s3_devkitm/esp32s3/procpu samples/drivers/ipm/ipm_esp32/ -p --sysbuild
+west flash && west espressif monitor
+```
+
+Three images has been created. Image for the PROCPU is called `ipm_esp32`. Image for the APPCPU is called `ipm_esp32_remote`. Finally MCUboot image (run by PROCPU) is called `mcuboot`.
+
+After flashing and connecting to a target using the serial port you should be able to observe following output on the console.
```

Comment:
Is it worth providing instructions or some reminders here on how to flash and connect to the serial port? If the readers are supposed to know it already, then please ignore my question.

```suggestion
After flashing and connecting to a target using the serial port, we should be able to see the following output in the console:
```

Reply:
the flash instructions are part of the code block describing a build